packages/components/src/components/NFTGallery/NFTGallery.stories.tsx

```diff
@@ -30,10 +30,10 @@ const nftsOwnedByAnENSStory = () => {
 };
 
 export const nftsOwnedByAnENS = nftsOwnedByAnENSStory.bind({});
-// @ts-expect-error
+// @ts-expect-error attaching parameters is fine when binding stories
```

Comment:
`@ts-expect-error` directives should have a description so others can understand why an error is expected.